crm_rma_stock_location/__init__.py

```diff
@@ -19,5 +21,14 @@
 #
 ##############################################################################
 
-from . import stock_warehouse
-from . import product
+from . import models
+from . import wizards
+from openerp import SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
```

Comment:
@moylop260 I think this is probably something extra to add for directory module structure, I'll propose that later, what do you think? I'm going to proceed to do the refactor you pointed out.